src/xrScriptEngine/script_engine.cpp

```diff
@@ -754,7 +754,12 @@ void CScriptEngine::lua_error(lua_State* L)
 
 int CScriptEngine::lua_pcall_failed(lua_State* L)
 {
-    print_output(L, "", LUA_ERRRUN);
+#if (!defined(DEBUG) && !XRAY_EXCEPTIONS)
+    print_output(L, "", 0); // xxx: Force game to not break in raii_guard() for this type of errors
```

Comment:
XXX comments are used to show that something has to be done later. Do not use XXX when you just explain what's going on.